setup.sh

```diff
@@ -21,6 +21,7 @@ sudo apt-get install python-psycopg2 --yes
 sudo apt-get install python-gobject --yes
 sudo apt-get install python-pycurl --yes
 sudo apt-get install postgresql-9.3-postgis-2.1 --yes
+sudo apt-get install landscape-client --yes
```

Comment:
I think whether we install this package still needs more research. I don't see why it's relevant to our functionality.

Take a look at
https://launchpad.net/ubuntu/+source/landscape-client

Landscape is Canonical's server monitoring and management application.